Mage.Sets/src/mage/cards/i/IanTheReckless.java

```diff
@@ -0,0 +1,62 @@
+package mage.cards.i;
+
+import mage.MageInt;
+import mage.abilities.TriggeredAbility;
+import mage.abilities.common.AttacksTriggeredAbility;
+import mage.abilities.condition.Condition;
+import mage.abilities.condition.common.SourceMatchesFilterCondition;
+import mage.abilities.decorator.ConditionalInterveningIfTriggeredAbility;
+import mage.abilities.dynamicvalue.common.SourcePermanentPowerCount;
+import mage.abilities.effects.common.DamageControllerEffect;
+import mage.abilities.effects.common.DamageTargetEffect;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.constants.CardType;
+import mage.constants.SubType;
+import mage.constants.SuperType;
+import mage.filter.FilterPermanent;
+import mage.filter.predicate.permanent.ModifiedPredicate;
+import mage.target.common.TargetAnyTarget;
+
+import java.util.UUID;
+
+/**
+ * @author notgreat
+ */
+public final class IanTheReckless extends CardImpl {
+
+    private static final FilterPermanent filter = new FilterPermanent();
+
+    static {
+        filter.add(ModifiedPredicate.instance);
+    }
+
+    private static final Condition condition = new SourceMatchesFilterCondition(filter);
+
+    public IanTheReckless(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[]{CardType.CREATURE}, "{1}{R}");
+
+        this.supertype.add(SuperType.LEGENDARY);
+        this.subtype.add(SubType.HUMAN);
+        this.subtype.add(SubType.WARRIOR);
+        this.power = new MageInt(2);
+        this.toughness = new MageInt(1);
+
+        // Whenever Ian the Reckless attacks, if it's modified, you may have it deal damage equal to its power to you and any target.
```

Comment:
Looks like any cards with `ModifiedPredicate` usage can have `ConditionHint` for better UX.

Reply:
That's generally used for "if you control a modified creature".

Reply:
yeah, I don't think a hint is really needed here